README.md

```diff
@@ -90,14 +90,38 @@ pip install -U https://github.com/pygfx/pygfx/archive/main.zip
 Under development, many things can change.
 
 
-## Example testing
+## Testing examples
 
-Include the comment `# test_example = true` in an example to have pytest run it as part of the test suite.
+There are two types of tests for examples included with pygfx:
 
-To support testing an example, ensure the following requirements are met:
+### Type 1: Checking if examples can run
+
+When running the test suite with `--slow` (enabled on CI), pytest will run every example in a subprocess,
+to see if it can run and exit cleanly. You can opt out of this mechanism by including the comment
+`# run_example = false` in the module.
+
+### Type 2: Checking if examples output an image
+
+You can also (independently) opt-in to output testing for examples, by including the comment
+`# test_example = true` in the module. Output testing means the test suite will
+attempt to import the `canvas` instance global from your example, and call `draw()` on it
```

Comment:
Mmm, we're also hiding the canvas behind the renderer since #275. Can we use the global renderer instead?

Reply:
Well, I'm specifically relying on the OffscreenCanvas-only `draw()` method, which is not available on the Renderer. So, I'm not sure, what do you think?

Reply:
```py
im = example.renderer.target.draw()
```

Reply:
Done!